Listener/RequestListener.php

```diff
@@ -1,5 +1,7 @@
 <?php
 
+declare(strict_types=1);
```

Comment:
Could be defined in .php_cs file ? (Same for private methods)

Reply:
What do you mean? Do Travis will fail if we do not have this line?

Reply:
Yes m'y point is to switch all files to strict. And add the rule in php_cs to not forget it.

Reply:
👍  Please send a PR with that